client/src/main/java/com/alibaba/nacos/client/config/impl/ClientWorker.java

```diff
@@ -275,39 +277,39 @@ private void checkLocalConfig(CacheData cacheData) {
         final String dataId = cacheData.dataId;
         final String group = cacheData.group;
         final String tenant = cacheData.tenant;
-        File path = LocalConfigInfoProcessor.getFailoverFile(agent.getName(), dataId, group, tenant);
+        File path = LocalConfigInfoProcessor.getFailoverFile(this.agent.getName(), dataId, group, tenant);
 
         // 没有 -> 有
```

Comment:
Translate the Chinese annotation into English

Reply:
Okay, I will resolve original issue here.